bedwars-plugin/src/main/java/com/tomkeuper/bedwars/listeners/DamageDeathMove.java

```diff
@@ -542,21 +543,33 @@ public void onRespawn(PlayerRespawnEvent e) {
     public void onMove(PlayerMoveEvent e) {
         if (Arena.isInArena(e.getPlayer())) {
             IArena a = Arena.getArenaByPlayer(e.getPlayer());
+            int updateDistance = 50; // update distance measured in blocks
```

Comment:
Update distance? should this be configurable? what does it do?

Reply:
Update distance is the distance where the holograms should be updated from.

There are some pros and cons about making it configurable.
Which arent much of a problem so yes we should make it configurable but we should explain it in the config or somewhere else